packages/docs/content/components/drawer.mdx

```diff
@@ -78,36 +78,38 @@ This component has analytics tracking available. Please see our developer docume
+- When optional information is needed to complete a task without overwhelming the primary content on the page.
 
-### When to consider alternatives
+`HelpDrawer`
+- When you need to track how users interact with the content, such as monitoring how often the drawer is opened or engaged with.
 
-- When the content is timely/urgent, or critical to the action being taken.
-- When the content is intended to clarify or call attention to other content on the page.
-- When the content must be displayed without being initiated by a user action.
-- When additional information is necessary to the primary information the user is interacting with.
+### When to Consider Alternatives
+
+- When the information is timely, urgent, or critical to the action being taken. Drawers are intended for supplementary details that users can access at their discretion. For time-sensitive or essential information, consider using more prominent inline elements to ensure users don’t miss it.
+- When the information needs to be immediately visible to clarify or support a primary task. Since `Drawers` require an extra step to open, they are not ideal for essential information that users must see to complete a task or understand other content.
+- When the information must be displayed automatically without user initiation. `Drawers` rely on users clicking a link or button to open them. If content needs to appear automatically, such as onboarding tips or error messages, consider inline options that don’t require user action.
 
 ### Usage
 
-- Link text used to trigger the help drawer should be specific and descriptive.
-  - Avoid using only "Learn more" to link to help drawer content
-  - Limit the amount of link text
-- Drawer content area should include a close button clearly at the top.
+- Link text used to trigger the `Drawer` should be specific and descriptive.
+  - Avoid using only "Learn more" to link to help `Drawer` content.
+  - Limit the amount of link text.
+- `Drawer` content area should include a close button clearly at the top.
+- On larger screens, the `Drawer` is fixed to the side of the screen, whereas on smaller screens, it overlays the entire screen.
 
 ### Accessibility
 
-- Drawer should have an `aria-labelledby` attribute that references the `id` of heading in the Drawer.
-- The heading level used within the Drawer should fit semantically with the heading levels of the rest of the page. For instance, if the heading level before the Drawer toggle link is an `<h2>`, the heading level of the Drawer would most likely be an `<h3>`.
+- The `Drawer` should have an `aria-labelledby` attribute that references the `id` of the heading in the `Drawer`.
+- The `Drawer` should be rendered below the toggle button that triggers it. This way, the markup remains semantically sound and accessible for screen readers, improving the user experience.
```

Comment:
I'm not sure if all audience members will understand "rendered below the toggle button". I had to read it a couple times to come to the conclusion that what I think it means is that the markup for the drawer should go after the toggle in the DOM. Is that what it's saying? It's an HTML thing not a visual thing?

Reply:
Thanks for the feedback! The original bullet was worded like this: "Render the drawer below the toggle button that triggers it. This way the markup remains semantically sound and screen reader friendly." I took that to mean it was a semantic HTML thing rather than a visual positioning thing, but I can see how the new wording could come across as unclear.

What do you think about rephrasing it like this?
```
- The `Drawer` markup should be placed below the toggle element that triggers it in the HTML structure. This keeps the markup semantically sound and improves accessibility for screen readers.
```

Reply:
Agree!